### PR TITLE
Create environment for frontend

### PR DESCRIPTION
**Pull request by mikulus8922**
Add Dockerfile.frontend

### CODE REVIEW

File: docker/Dockerfile.frontend [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+
+FROM node:16.14.0-buster
+ENV NODE_ENV=production
+
+WORKDIR /app
+
+COPY ["frontend/package.json", "frontend/package-lock.json*", "./"]
```
Comment (kkuriata) [inline docker/Dockerfile.frontend:8]: Whole app could be copied in one entry, because later you copy it anyways

File: docker/Dockerfile.frontend [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
```
Comment (kkuriata) [inline docker/Dockerfile.frontend:1]: This is not needed

**Review by kkuriata — CHANGES_REQUESTED** [review]

**Review by mwidera — APPROVED** [review]
Please just fix the Dockerfile comments from @kkuriata 

I'm going to approve, as I've nothing more to add here

File: docker/Dockerfile.frontend (hunk 1 above)
Comment (mikulus8922) [reply docker/Dockerfile.frontend]: Done

File: docker/Dockerfile.frontend (hunk 2 above)
Comment (mikulus8922) [reply docker/Dockerfile.frontend]: Done

**Review by kkuriata — APPROVED** [review]

**Pull request merged**